php-classes/Slate/Connectors/GoogleSheets/Connector.php

```diff
@@ -100,4 +107,116 @@ public static function synchronize(IJob $Job, $pretend = true)
 
         return true;
     }
+
+    public static function pullStudents(IJob $Job, SpreadsheetReader $spreadsheet, $pretend = true)
```

Comment:
why did we override this whole method instead of editing the parent class? There was no intention for this behavior change to _only_ affect google sheets, it can affect all spreadsheet impots

Reply:
noted. will close open PR upstream.